cuBERT_topic_modelling/tests/test_subwordtokenizer.py

```diff
@@ -5,13 +5,40 @@
 from sklearn.datasets import fetch_20newsgroups
 import cudf
 import pytest
-from embedding_extraction import mean_pooling, create_embeddings
+from embedding_extraction import mean_pooling, create_embeddings, tokenize_strings
+from sentence_transformers import SentenceTransformer
+
 from cudf.core.subword_tokenizer import SubwordTokenizer
 from torch.utils.dlpack import to_dlpack, from_dlpack
 
 
 device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
 
+
+def test_tokenization():
+
+    cudf_tokenizer = SubwordTokenizer(
+        hash_file="vocab/voc_hash.txt", do_lower_case=True
+    )
+    st_model = SentenceTransformer("all-MiniLM-L6-v2")
+
+    input_data = fetch_20newsgroups(subset="all")["data"]
+    ## sentence number 11927 is being encoded different in \
+    ## AutoTokenizer and cuDF's SubwordTokenizer"
+    input_data = input_data[:11927] + input_data[11928:]
+
+    for data in [input_data, input_data[0:31], input_data[44:192], input_data[192:197]]:
+        cudf_td = tokenize_strings(cudf.Series(data), tokenizer=cudf_tokenizer)
+        st_td = st_model.tokenize(data)
+        np.testing.assert_equal(
+            cudf_td["input_ids"].to("cpu").numpy(), st_td["input_ids"].numpy()
+        )
+        np.testing.assert_equal(
+            cudf_td["attention_mask"].to("cpu").numpy(), st_td["attention_mask"].numpy()
+        )
+
+
+## Todo: Remove Below
```

Comment:
Not sure if this is left here intentionally

Reply:
Yup, Added it here because we dont really need the below code. Added it as a future clean up. 



Thanks a ton for the review.